lfs/progress_meter.go

```diff
@@ -212,3 +214,58 @@ func formatBytes(i int64) string {
 
 	return fmt.Sprintf("%d B", i)
 }
+
+// Indeterminate progress indicator 'spinner'
+type Spinner struct {
+	stage int
+	msg   string
+}
+
+var spinnerChars = []byte{'|', '/', '-', '\\'}
+
+// Print a spinner (stage) to out followed by msg (no linefeed)
+func (s *Spinner) Print(out io.Writer, msg string) {
+	s.msg = msg
+	s.Spin(out)
+}
+
+// Just spin the spinner one more notch & use the last message
+func (s *Spinner) Spin(out io.Writer) {
+	s.stage = (s.stage + 1) % len(spinnerChars)
+	s.update(out, string(spinnerChars[s.stage]), s.msg)
+}
+
+// Finish the spinner with a completion message & newline
+func (s *Spinner) Finish(out io.Writer, finishMsg string) {
+	s.msg = finishMsg
+	s.stage = 0
+	var sym string
+	if runtime.GOOS == "windows" {
+		// Windows console sucks, can't do nice check mark except in ConEmu (not cmd or git bash)
+		// So play it safe & boring
+		sym = "Done"
+	} else {
+		sym = fmt.Sprintf("%c", '\u2714')
```

Comment:
Looks like this will output "Done 3 objects found". Here's what I see:

```
$ git lfs fetch --all    
Scanning for all objects ever referenced...
✔ 3 objects found 
```

Something like `*` might be just fine.